app/models/deploy.rb

```diff
@@ -42,16 +40,23 @@ def job_execution_queue_name
     "stage-#{stage.id}" unless stage.run_in_parallel
   end
 
-  def summary
-    "#{job.user.name} #{deploy_buddy} #{summary_action} #{short_reference} to #{stage.name}"
+  # job has almost identical code, keep it in sync
+  def summary(global: false)
```

Comment:
I don't understand what `global` means in this context.  Better name for variable?